server/schema/rootQueryType.js

```diff
@@ -2,12 +2,46 @@ const axios = require('axios');
 const { GraphQLObjectType, GraphQLInt, GraphQLList } = require('graphql');
 const GameType = require('./types/GameType');
 const PlayerType = require('./types/PlayerType');
+const CreatorType = require('./types/CreatorType');
+const OpponentType = require('./types/OpponentType');
 
 const RootQuery = new GraphQLObjectType({
-    name: 'RootQueryType',
-    fields: {
-      getGame: {
-          type: GameType,
+  name: 'RootQueryType',
+  fields: {
+    getCreator: {
+      type: CreatorType,
+      args: { id: { type: GraphQLInt } },
+      resolve(parentValue, args) {
+        console.log('getcreator',parentValue, args)
+        return axios.get(`http://localhost:1337/creators/${args.id}`)
```

Comment:
Would be nice if the URL was fetched from a config-file, so there is no dependency on localhost.

Perhaps add a config.js.sample?